pkg/ui/src/util/eventTypes.ts

```diff
@@ -33,11 +33,16 @@ export const FINISH_SCHEMA_CHANGE = "finish_schema_change";
 export const NODE_JOIN = "node_join";
 // Recorded when an existing node rejoins the cluster after being offline.
 export const NODE_RESTART = "node_restart";
+// Recorded when a node is marked as decommissioning.
+export const NODE_DECOMMISSIONED = "node_decommissioned";
+// EventLogNodeRecommissioned is recorded when a decommissioned node is
```

Comment:
s/EventLogNodeRecommissioned is recorded/Recorded/

Reply:
Thanks. Done.